Tasks/quotes.js

```diff
@@ -1,30 +1,23 @@
 // Change double quotation to open or close quotation
+'use strict';
 
-const EMPTY = '';
-
-quotes = function (s) {
-  res = [];
-  open = false;
-  for (c of s) {
-    if (c === '"') {
-      for (i of c) {
-        if (!open) {
-          res.push('«');
-          open = true;
-        } else {
-          res.push('»');
+const quotes = function (inputString) {
```

Comment:
Prefer arrow-functions
```suggestion
const quotes = (inputString) => {
```